packages/common/src/utils.ts

```diff
@@ -397,3 +397,39 @@ export function intToBigInt(value: IntegerType, signed: boolean): bigint {
 export function with0x(value: string): string {
   return !value.startsWith('0x') ? `0x${value}` : value;
 }
+
+export function hexToBigInt(hex: string): bigint {
+  if (typeof hex !== 'string')
+    throw new TypeError('hexToNumber: expected string, got ' + typeof hex);
+  // Big Endian
+  return BigInt(`0x${hex}`);
+}
```

Comment:
Can you add a docs comment to the function specifying that the `hex` input is supplied without the `0x` prefix and in big endian?

Reply:
This was marked as resolved but doesn't appear to be resolved